### ISSUE TITLE
/etc/sysctl.d/trident-core.conf entries not processed

### ISSUE THREAD

**Issue by jimbygh** [project-trident/trident-core#53]
sysctl entries in /etc/sysctl.d/trident-core.conf  are not processed during startup.

currently not working on 18.11-PRERELEASE_2

steps to reproduce:
1. to see just the requested sysctls in that file:
```
cd /etc/sysctl.d
cat trident-core.conf | grep -v "^#" | sort -u | awk '{print $1}'
```
2. to see the actual values on a running system:
```
cd /etc/sysctl.d
for i in `cat trident-core.conf | grep -v "^#" | sort -u | awk -F '=' '{print $1}'`
do
sysctl $i
done
```

**Comment by jimbygh** [comment]
ok, in /etc/init.d/sysctl, in function BSD_sysctl(),  the line at the top of the function

[ -e /etc/sysctl.conf ] || return 0

says, if /etc/sysctl.conf does not exist, return zero and do not process the rest of the file, meaning do not process any /etc/sysctl.d/*.conf entries.

this is an openrc dev issue, but can be patched, but just adding an empty /etc/sysctl.conf during overlay time.

i'll update the ticket with this detail

**Comment by beanpole135** [comment]
Trident has that file existance check already hot-patched for future checks on bootup:
https://github.com/project-trident/trident-core/blob/master/core-files/usr/local/share/trident/scripts/system-sanity-check.sh

Should be fixed in 18.12-PRERELEASE_2 or later.
